packages/lib/shim-init-node.js

```diff
@@ -186,36 +186,37 @@ function shimInit(options = null) {
 			const nativeImage = require('electron').nativeImage;
 			let image = nativeImage.createFromPath(filePath);
 			if (image.isEmpty()) throw new Error(`Image is invalid or does not exist: ${filePath}`);
-
 			const size = image.getSize();
 
-			let mustResize = size.width > maxDim || size.height > maxDim;
+			const canResize = size.width > maxDim || size.height > maxDim;
```

Comment:
I changed it from mustResize to canResize to be more semanticly correct, in my opinion, since when we use MUST in english it means an obligation. As the user has the possibility rather resizing the image or not, "can" seems to be more correct to me.

Reply:
I agree. Also, thank you for changing `let` to `const` :).